src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -190,6 +189,39 @@ public int getRollupCountForCriteriaGroups(
         .get(0);
   }
 
+  private SnapshotBuilderDomainOption getDomainOptionFromSettingsByName(
```

Comment:
This API will be changing soon with the work to use domain number IDs instead of domain names, so maybe it's not important to change it now. And this name does match up with the OMOP terms; the `domain` table has `domain_id` which is a string like `Condition` and it also has `domain_concept_id` which is an integer key in the `concept` table. https://ohdsi.github.io/CommonDataModel/cdm54.html#domain